### PR TITLE
Convert samplesheet to value channel

### PR DESCRIPTION
**Pull request by nschcolnicov**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/nanostring/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/nanostring _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### `Fixed`
 
+- [#117](https://github.com/nf-core/nanostring/pull/117) - Fix input samplesheet, convert it to value channel.
```
Comment (christopher-mohr) [inline CHANGELOG.md:20]:
```suggestion
- [#119 ](https://github.com/nf-core/nanostring/pull/119) - Fix input samplesheet, convert it to value channel. [#118](https://github.com/nf-core/nanostring/issues/118)
```

**Review by christopher-mohr — CHANGES_REQUESTED** [review]
Thank you for the quick fix! One minor thing in the `CHANGELOG`, otherwise LGTM.

**Review by christopher-mohr — APPROVED** [review]
LGTM!

**Comment by christopher-mohr** [comment]
Do we need both versions of the gene heatmap file (the one including `wo_HKnorm_`) for the tests? But I thought the tests were passing before?

**Comment by nschcolnicov** [comment]
@christopher-mohr There seems to be a random behavior in the tests when they are run in github actions where a suffix gets added to the files, making them sometimes fail. 
Running them a second time will ensure that they pass, but I created a ticket for this: https://github.com/nf-core/differentialabundance/issues/376.

**Comment by atrigila** [comment]
Perhaps you need to apply a similar solution like @alanmmobbs93 did here: https://github.com/nf-core/nanostring/blob/fad81c8cf68163de556e43f4515f179b4b21c2c7/workflows/nanostring.nf#L98

**Pull request merged**